File: abc/abc359/abc359_a/abc359_a.py
n = int(input())
# a = list(map(int, input().split()))
# x, y = map(int, input().split())
# s = input()

ans = 0
for _ in range(n):
  s = input()
  if s == 'Takahashi':
    ans += 1

print(ans)

File: abc/abc359/abc359_a/abc359_a.rs
use proconio::input;

fn main() {
    input! {
        n: u8,
    }

    let mut ans = 0;
    for _ in 0..n {
        input! {
            s: String,
        }
        if s == "Takahashi" {
            ans += 1;
        }
    }

    print!("{}", ans);
}